### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by alroniks**
- [ ] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by mlinksva** [comment]
Just in case there's anything useful in them, there have been a few previous starts at Russian translations. #1299 has a couple articles translated (most of the files in that PR are still English).

**Comment by alroniks** [comment]
@mlinksva Thank you for the information. I will have a look and if possible try to fetch existing translations.
I saw that it was closed due to inactivity and it is sad, but it happens in open source sometimes, unfortunately. 

I have a personal goal, during translation learn articles more carefully, so I hope my attempt will be successful. But who knows? :)

File: _data/locales/ru.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ru:
+  locale_name: Русский
+  nav:
+    about: О проекте
+    contribute: Сотрудничество
+  index:
+    lead: Программы с открытым кодом создаются такими же людьми, как и вы. Узнайте, как запустить и развить свой проект.
```
Comment (mobilase) [inline _data/locales/ru.yml:7]: May be change "развить" to "развивать"? For me it seems more literaturaly.

File: _data/locales/ru.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    back_to_all_guides: Назад на главную
+    related_guides: Связанные темы
+  footer:
+    contribute:
+      heading: Сотрудничество
+      description: Хотите что-то предложить? Эти материалы доступны для редактирования. Помогите нам их улучшить.
+      button: Сотрудничать
+    subscribe:
+      heading: Быть в курсе
+      description: Узнавайте первыми о свежих заметках и советах об Open Source от GitHub.
```
Comment (mobilase) [inline _data/locales/ru.yml:20]: May be change to "Узнавайте первым о свежих советах и ресурсах GitHub с открытым исходным кодом"?

File: _articles/ru/index.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+layout: index
+title: Как делать Open Source
```
Comment (mobilase) [inline _articles/ru/index.html:3]: May be change to "Как работать с открытым исходным кодом"?

**Review by jaceppo — APPROVED** [review]

**Review by TaMaMa69 — APPROVED** [review]
Bhj

**Comment by vuchien000** [comment]
Iio

File: _data/locales/ru.yml (hunk 1 above)
Comment (MaryJaneLove4) [reply _data/locales/ru.yml:7]: Am lost

**Comment by noncash15** [comment]
alroniks:ru-translation

**Review by pipecc — APPROVED** [review]

File: _data/locales/ru.yml (hunk 2 above)
Comment (MaryJaneLove4) [reply _data/locales/ru.yml:20]: Yes

File: _articles/ru/index.html (hunk 3 above)
Comment (MaryJaneLove4) [reply _articles/ru/index.html:3]: (*_*)

**Review by MaryJaneLove4 — APPROVED** [review]

**Comment by MaryJaneLove4** [comment]
Czech translator

**Comment by MaryJaneLove4** [comment]
Dekuji

**Review by MaryJaneLove4 — APPROVED** [review]
It's right.

**Comment by MaryJaneLove4** [comment]
👍

**Review by Budda0ne — COMMENTED** [review]
[go1.16.3.src.tar.gz](https://github.com/github/opensource.guide/files/6980065/go1.16.3.src.tar.gz)

**Review by oneemo95 — APPROVED** [review]

**Review by KashcheyMC — COMMENTED** [review]
[](https://www.google.com/url?sa=t&source=web&rct=j&url=https://translate.google.com/%3Fhl%3Dru&ved=2ahUKEwiJ17DWnrDyAhWQO-wKHWD4CjgQFnoECBIQAg&usg=AOvVaw1GeJFpbqMHiaaRur8045v7)

**Comment by Aparus** [comment]
> @mlinksva Thank you for the information. I will have a look and if possible try to fetch existing translations.
> I saw that it was closed due to inactivity and it is sad, but it happens in open source sometimes, unfortunately.
> 
> I have a personal goal, during translation learn articles more carefully, so I hope my attempt will be successful. But who knows? :)

Привет. Переводишь ещё? :-) я ещё несколько статей перевёл. В прошлый раз не разобрался как тесты запустить и т.д. Они потом добавили более подробное руководство.

**Review by Mohsen-Phoenix — APPROVED** [review]

**Review by se7enhl — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
@alroniks What's the latest status here? Please try to merge `main` into your branch so CI is run. Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Need to fix CI failures: https://github.com/github/opensource.guide/pull/2395/checks?check_run_id=3532336781#step:7:37

**Comment by MikeMcQuaid** [comment]
Russian translation merged in https://github.com/github/opensource.guide/pull/2513.

**Pull request closed without merging**